eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java

```diff
@@ -254,5 +270,15 @@ private boolean removePackages() {
             return changed;
         }
 
+        private boolean showNewControls() {
```

Comment:
I would prefer to name the method isVersion21OrHigher. In few days, these controls won't be really new anymore :-)